hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterRpcServices.java

```diff
@@ -2424,14 +2425,14 @@ private RegionInfo getRegionInfo(HBaseProtos.RegionSpecifier rs) throws UnknownR
   public MasterProtos.AssignsResponse assigns(RpcController controller,
       MasterProtos.AssignsRequest request)
     throws ServiceException {
-    LOG.info(master.getClientIdAuditPrefix() + " assigns");
     if (this.master.getMasterProcedureExecutor() == null) {
       throw new ServiceException("Master's ProcedureExecutor not initialized; retry later");
     }
     MasterProtos.AssignsResponse.Builder responseBuilder =
         MasterProtos.AssignsResponse.newBuilder();
     try {
       boolean override = request.getOverride();
+      LOG.info("HbckService: {} assigns, override={}", master.getClientIdAuditPrefix(), override);
```

Comment:
Do you need the 'HbckService' prefix? Only it can call the assigns method, no?

Reply:
Remove "HbckService" to keep same with others log style.